### ISSUE TITLE
repeated <stream_info name="rof.iaf_jra"> in drof.streams.xml

### ISSUE THREAD

**Issue by aekiss** [COSIMA/om3-scripts#29]
[generate_xml_drof.py](https://github.com/COSIMA/om3-scripts/blob/main/data_stream_xml_generation/generate_xml_drof.py) uses
```xml
<stream_info name="rof.iaf_jra">
```
for both `friver` and `licalvf`. Presumably this is a bad idea?

Related: https://github.com/COSIMA/om3-scripts/issues/27

**Comment by aekiss** [comment]
Non-uniqueness bugs like these can be avoided if a dict is used instead of an array [here](https://github.com/COSIMA/om3-scripts/blob/main/data_stream_xml_generation/generate_xml_drof.py#L55-L58), with the stream name being the dict key.

**Comment by dougiesquire** [comment]
Closed by #31
